Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:9.2.0-alpine
+FROM node:10.13.0-alpine
```

Comment:
should be node 8

Reply:
`9.2.0` was the previously used  - let it stay the same